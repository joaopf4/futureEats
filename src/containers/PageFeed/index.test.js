import React from 'react'
import renderer from 'react-test-renderer'
import {
    FeedWrapper, LogoWrapper, Logo,
    NavBar, OptionText, ContentWrapper
} from '../PageFeed/style'

test("Testando styled do FeedWrapper", () => {
    const newSnapShot = renderer.create(<FeedWrapper />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do LogoWrapper", () => {
    const newSnapShot = renderer.create(<LogoWrapper />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do Logo", () => {
    const newSnapShot = renderer.create(<Logo />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do NavBar", () => {
    const newSnapShot = renderer.create(<NavBar />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do OptionText", () => {
    const newSnapShot = renderer.create(<OptionText />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})
test("Testando styled do ContentWrapper", () => {
    const newSnapShot = renderer.create(<ContentWrapper />).toJSON()

    expect(newSnapShot).toMatchSnapshot();
})