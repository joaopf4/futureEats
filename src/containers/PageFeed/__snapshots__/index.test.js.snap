// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando styled do ContentWrapper 1`] = `
<div
  className="sc-EHOje dfZaHS"
/>
`;

exports[`Testando styled do FeedWrapper 1`] = `
<div
  className="sc-bdVaJa cBeEWh"
/>
`;

exports[`Testando styled do Logo 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-htpNat hGNebm"
/>
`;

exports[`Testando styled do LogoWrapper 1`] = `
<div
  className="sc-bwzfXH dxaQdq"
/>
`;

exports[`Testando styled do NavBar 1`] = `
<div
  className="sc-bxivhb gzmmVO"
/>
`;

exports[`Testando styled do OptionText 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10 sc-ifAKCX bmRnqm"
/>
`;
